src/richText.js

```diff
@@ -456,39 +459,8 @@ define([
      * Wrap top-level expression nodes with bubble markup
      */
     function bubbleExpression(text, form) {
-        var el = $('<div>').html(text);
-        var EXPR = form.xpath.models.XPathInitialContextEnum.EXPR,
-            ROOT = form.xpath.models.XPathInitialContextEnum.ROOT,
-            expr = new logic.LogicExpression(text, form.xpath),
-            // Uses top level paths, because filters should not be made to bubbles
-            paths = _.chain(expr.getTopLevelPaths())
-                .filter(function(path) {
-                    var context = path.initial_context,
-                        numFilters = _.reduce(path.steps, function(memo, step) {
-                           return memo + step.predicates.length;
-                        }, 0),
-                        hasSession = /commcaresession/.test(path.toXPath());
-
-                    if (context === EXPR && (numFilters > 1 || !hasSession) ||
-                        context === ROOT && numFilters > 0) {
-                        return false;
-                    }
-
-                    return true;
-                })
-                .map(function(path) { return path.toXPath(); })
-                .uniq().value().concat(_.map(expr.getHashtags(), function(hashtag) {
-                    return hashtag.toHashtag();
-                }));
-
-        _.each(paths, function(path) {
-            var newPath = replacePathWithBubble(form, path);
-            el.html(el.html().replace(
-                new RegExp(RegExp.escape(path).replace(/ /g, '\\s*'), 'mg'),
-                $('<div>').append(newPath).html()
-            ));
-        });
-        return el.html();
+        text = form.normalizeBanana(text).replace('<', '&lt;').replace('>', '&gt;');
```

Comment:
I'm not sure what's supposed to come out of `normalizeBanana()`, but this is a naive way to escape XML. I think `xml.normalize()` may be what you want.